app/javascript/components/Decisions.jsx

```diff
@@ -0,0 +1,33 @@
+import isEmpty from "ramda/src/isEmpty";
```

Comment:
@SerhiiBorozenets Might be we should use some helper function from the `app/javascript/components/helpers/helper.js` or move ramda helpers functions in that helper?